### PR TITLE
Calico VPP docs cleanup for GA

### PR DESCRIPTION
**Pull request by onong**
Product Version(s):
v3.27.0

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico/operations/troubleshoot/vpp.mdx [hunk 1]
```diff
@@ -113,23 +112,23 @@ calivppctl vppctl myk8node1
 To list existing interfaces and basic counters use
 
 ```
```
Comment (ctauchen) [inline calico/operations/troubleshoot/vpp.mdx:114]:
```suggestion
```bash
```

File: calico/operations/troubleshoot/vpp.mdx [hunk 2]
```diff
@@ -113,23 +112,23 @@ calivppctl vppctl myk8node1
...
-vpp show int
-vpp show int addr
+vpp# show int
```
Comment (ctauchen) [inline calico/operations/troubleshoot/vpp.mdx:115]: This looks like an odd change. Did the command name change to `vpp#`?

File: calico/operations/troubleshoot/vpp.mdx (hunk 1 above)
Comment (ctauchen) [reply calico/operations/troubleshoot/vpp.mdx:114]: This should be done throughout.

File: calico/operations/troubleshoot/vpp.mdx [hunk 3]
```diff
@@ -139,60 +138,64 @@ vpp show buffers   # You should have non zero free buffers, otherwise traffic wo
...
- If you want more info about this interface (name in Linux, queues, descriptors, ...)
-vpp show tun tun3
-Interface: tun3 (ifindex 5)
-  name "eth0"
-  host-ns "/proc/17675/ns/net"
+Container interfaces are named `tun[0-9]+`. You can find which one belong to which container as follows:
+
+- Connect to VPP
+  ```
+  $ calivppctl vppctl NODENAME
```
Comment (ctauchen) [inline calico/operations/troubleshoot/vpp.mdx]: Remove prompt at start of bash command.

**Review by ctauchen — COMMENTED** [review]
Thanks for this! I have a few small changes for now, and I'll have another look after @sridhartigera has done his technical review.

**Comment by sridhartigera** [comment]
> Thanks for this! I have a few small changes for now, and I'll have another look after @sridhartigera has done his technical review.

@ctauchen I don't think I am the right person to do the technical review. I am going to test it once the docs are in,

File: calico/operations/troubleshoot/vpp.mdx (hunk 2 above)
Comment (onong) [reply calico/operations/troubleshoot/vpp.mdx:115]: "vpp#" is VPP's cli prompt and I have just rectified the mistake where the "#" was missing :)

File: calico/operations/troubleshoot/vpp.mdx (hunk 1 above)
Comment (onong) [reply calico/operations/troubleshoot/vpp.mdx:114]: Doing so would **italicize** the text after the "vpp#" and thats why I have used "bash" in some places and not used in some other places.

File: calico/operations/troubleshoot/vpp.mdx (hunk 3 above)
Comment (onong) [reply calico/operations/troubleshoot/vpp.mdx]: Done

**Review by sknat — COMMENTED** [review]
Thanks @onong, looks good to me !

**Comment by ctauchen** [comment]
LGTM and merging

**Pull request merged**